src/components/ebay-menu-button/examples/12-icon-without-toggle/template.marko

```diff
@@ -3,7 +3,7 @@
  * Below we have added an empty class to force this to be the case.
  */
 class {};
-<ebay-menu-button icon="mic" a11y-text="Microphone" no-toggle-icon>
+<ebay-menu icon="settings" a11y-text="Settings" no-toggle-icon>
```

Comment:
Oh, this changes from `<ebay-menu-button ... >` to `<ebay-menu ... >`, which is probably not correct.

Reply:
Good catch!